src/shared/bot_nav_shared.h

```diff
@@ -219,12 +219,15 @@ struct LinearAllocator : public dtTileCacheAlloc
 
 	void resize( const size_t cap )
 	{
-		if ( buffer )
+		auto * tmp_buffer = static_cast<unsigned char*>( realloc( buffer, cap ) );
+		if ( !tmp_buffer )
 		{
-			free( buffer );
+			//TODO: I doubt a Log::Warn is enough for that kind of situation,
```

Comment:
`Sys::Drop` would be good

Reply:
So be it, then!
Didn't knew about this one, I was naively searching for "Log::Error" or something along this line...